mlflow/openai/__init__.py

```diff
@@ -166,7 +166,7 @@ def get_openai_package_version():
 
 # See https://github.com/openai/openai-python/blob/cf03fe16a92cd01f2a8867537399c12e183ba58e/openai/__init__.py#L30-L38
 # for the list of environment variables that openai-python uses
-class OpenAIEnvVar(str, Enum):
+class _OpenAIEnvVar(str, Enum):
```

Comment:
Sphinx thinks this is a public class and documents it:

<img width="767" alt="image" src="https://user-images.githubusercontent.com/17039389/233283957-1950a744-edf9-4d4c-944d-ea6e6d156407.png">

https://mlflow.org/docs/latest/python_api/mlflow.openai.html

but it should not